generators/server/templates/src/main/docker/mssql.yml.ejs

```diff
@@ -27,7 +27,7 @@ services:
             - ACCEPT_EULA=Y
             - SA_PASSWORD=yourStrong(!)Password
             - MSSQL_DATABASE=<%= baseName %>
-            - MSSQL_SLEEP=7
+            - MSSQL_SLEEP=60
```

Comment:
Should we add the PID?  MSSQL_PID: "Developer" or  MSSQL_PID: "Express"

Reply:
MSSQL_PID=<your_product_id | edition_name> (default: Developer)

Reply:
default is `Developer`. If we want to change, then, I will prefer `Express` as it's free from any license obligations.

Reply:
don't hesitate to open a new PR if needed :)